search-app/src/cmr/search/site/data.clj

```diff
@@ -35,11 +35,11 @@
   request context which contains the state of a running CMR."
   :execution-context)
 
-(defmethod get-providers :cli
-  [context]
-  (let [providers-url (format "%sproviders"
-                              (config/application-public-root-url :ingest))]
-    (util/endpoint-get providers-url {:accept mt/json})))
+;; (defmethod get-providers :cli
+;;   [context]
```

Comment:
This commented out section should be deleted.

Reply:
I've actually uncommented this now; it's still used when generating static content via the command line. However, since the job scheduler now sets the ACLs manually, it has been changed no longer uses the cli mode.